ivy/functional/frontends/tensorflow/activations.py

```diff
@@ -12,3 +12,8 @@ def hard_sigmoid(x):
 
 def linear(x):
     return x
+
+def relu(x):

```

Comment:
I think there might be a cleaner way to do this, could we just not set the max value in `ivy.clip` to `ivy.inf`?

Reply:
Also, it seems that `x_max` is a required parameter on `ivy.clip`

Reply:
Yes but Relu essentially just keeps all positive values and set negative values to 0. So we could possibly try setting the x_max value to just ivy.inf?

Reply:
It wouldn't change the logic but it would just condense the implementation :)

Reply:
Hi, so Relu basically turns any negative values to 0 otherwise it just keeps them. So we could just set the x_max parameter in ivy.clip to ivy.inf.

Reply:
It would just reduce the implementation to one line of code:
`x = ivy.clip(x, 0.0, ivy.inf)`

Reply:
Hopefully that makes sense :)

Reply:
Yup, sounds good. I'll make the change by tomorrow, afk for the day sadly 😅

Reply:
Done, please check now

Reply:
Hi, the histories of this change was too divergent and showing merge conflicts, so I took the liberty of shifting to a new PR with merged changes at #5338